src/components/accordion/accordion.html

```diff
@@ -30,6 +30,8 @@
                 :large="iconSize=='large'"
                 :border="hasIconBorder"
                 :skin="plusSkin"
+                :title-open="titleMenuOpen"
+                :title-close="titleMenuClose"
                 aria-hidden="true"></m-plus>
```

Comment:
Respecter la convention des attr (svp revoir le ficher htlm au complet)

1 Attributs de contrôles (v-if, v-else, v-else-if, v-show, v-for avec :key)
2 id
3 class
4 :class
5 toutes les props regroupées ensemble (:xyz)
6 tous les événements regroupés ensemble (@xyz)
7 autre directives vue : v-xxx
8 autres attributs dans un ordre logique